src/quadpype/hosts/tvpaint/plugins/publish/extract_sequence.py

```diff
@@ -438,7 +439,7 @@ def render(
             mark_out (int): On which frame index export will end.
             layers (list): List of layers to be exported.
             ignore_layers_transparency (bool): Layer's opacity will be ignored.
-            apply_background (bool): Apply a bg color to the render set in settings of OP.
+            apply_bg_back_command (str): George script command to put the BG back as set in current scene.
```

Comment:
same, this should be simply a bool, not a george script command since we won't use it in this method